rmgpy/tools/canteramodel.py

```diff
@@ -497,19 +532,33 @@ def simulate(self):
                                                    )
                 condition_data.append(species_generic_data)
 
+            # save kinetic data as generic data object
             reaction_sensitivity_data = []
             for index, species in enumerate(self.sensitive_species):
                 for j in range(num_ct_reactions):
                     reaction_sensitivity_generic_data = GenericData(
                         label='dln[{0}]/dln[k{1}]: {2}'.format(species.to_chemkin(), j + 1, self.model.reactions()[j]),
                         species=species,
                         reaction=self.model.reactions()[j],
-                        data=sensitivity_data[:, num_ct_reactions * index + j],
+                        data=kinetic_sensitivity_data[:, num_ct_reactions * index + j],
                         index=j + 1,
                         )
                     reaction_sensitivity_data.append(reaction_sensitivity_generic_data)
 
-            all_data.append((time, condition_data, reaction_sensitivity_data))
+            # save thermo data as generic data object
+            thermodynamic_sensitivity_data = []
+            if self.thermo_SA:
+                for index, species in enumerate(self.sensitive_species):
+                    for j in range(num_ct_species):
+                        thermo_sensitivity_generic_data = GenericData(
+                            label='dln[{0}]/dG[{1}]'.format(species, self.model.species()[j].name),
```

Comment:
Should we also change here?

Reply:
I agree, we should be consistent either way